atomics/T1197/T1197.yaml

```diff
@@ -13,7 +13,7 @@ atomic_tests:
   executor:
     name: command_prompt
     command: |
-      bitsadmin.exe  /transfer /Download /priority Foreground https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/Windows/Execution/Bitsadmin.md $env:TEMP\AtomicRedTeam\bitsadmin_flag.ps1
+      bitsadmin.exe  /transfer /Download /priority Foreground https://raw.githubusercontent.com/redcanaryco/atomic-red-team/59b33d16d7ace0bfb32d12fbcf166cc2a6dce129/atomics/T1197/T1197.md $env:TEMP\AtomicRedTeam\bitsadmin_flag.ps1
```

Comment:
Are you intentionally locking this to a specific commit version of this file, or did you want "master" so you always have the latest?